src/styles/_article.scss

```diff
@@ -9,13 +9,21 @@
     h4,
     p {
       margin-top: 20px;
+      a {
+        color: black;
+        text-decoration: underline;
+      }
+      strong {
+        display: inline;
+        font-weight: bold;
```

Comment:
Sooo because the way I've loaded up the fonts we NEVER use `font-weight` in the Finsec project. Instead we only use `font-family: $bold` and `font-weight: $black` to define boldness of font.

MIND that if you DO declare `font-weight:bold` on top of our fonts, the browsers will add this really ugly bloat themselves. If bold fonts don't look crips or weird on our website, it's because it does not have `font-weight: normal`. Makes sense?

Check the `src/styles/_variables.sss` there you can see all the colours, but also the font families we assign to for example `$bold`.